packages/fxa-auth-server/lib/l10n/server.ftl

```diff
@@ -1,4 +1,5 @@
 ## Non-email strings
 
 session-verify-send-push-title = Logging in to { -product-firefox-accounts }?
+session-verify-send-push-title-2 = Logging in to { -product-mozilla-accounts }?
```

Comment:
```suggestion
session-verify-send-push-title-2 = Logging in to your { -product-mozilla-account }?
```

According to the marketing guielines around the rebrand, `account` is almost always singular and plural should only be used when referring to changes to all Mozilla accounts. So in this case we should go with singular.